tell/visualization.py

```diff
@@ -67,43 +67,43 @@ def plot_ba_service_territory(ba_to_plot: str, year_to_plot: str, data_input_dir
     ax1.set_title((ba_to_plot + ' Service Territory in ' + year_to_plot))
 
 
-def plot_state_scaling_factors(shapefile_input_dir: str, data_input_dir: str, year_to_plot: str, image_resolution: int,
-                               image_output_dir: str, save_images=0):
-    """Create state scaling factors map and save image in image directory
+def plot_state_scaling_factors(year_to_plot: str, data_input_dir: str, image_output_dir: str,
+                               image_resolution: int, save_images=False):
+    """Plot the scaling factor that force TELL annual total state loads to agree with GCAM-USA
 
-    :param shapefile_input_dir:        Directory where the Census TL shapefile is stored
-    :type shapefile_input_dir:         str
-
-    :param data_input_dir:             Directory to store the EIA 930 hourly load data as a csv
-    :type data_input_dir:              str
+    :param year_to_plot:        Year you want to plot (valid 2039, 2059, 2079, 2099)
+    :type year_to_plot:         str
 
-    :param year_to_plot:               What year to plot as a data visualization
-    :type year_to_plot:                str
+    :param data_input_dir:      Top-level data directory for TELL
+    :type data_input_dir:       str
 
-    :param save_images:                Save images to directory? 0 is no, 1 is yes (default = 0)
-    :type save_images:                 int
+    :param image_output_dir:    Directory to store the images
+    :type image_output_dir:     str
 
-    :param image_resolution:           Resolution in dpi to save images
-    :type image_resolution:            int
+    :param image_resolution:    Resolution at which you want to save the images in DPI
+    :type image_resolution:     int
 
-    :param image_output_dir:           Directory to store the image outputs
-    :type image_output_dir:            str
+    :param save_images:         Set to True if you want to save the images after they're generated
+    :type save_images:          bool
 
-    :return:                           State scaling factors map and save image in image directory
```

Comment:
@cdburley was the removal of all the return parameter descriptions in the docstrings intentional? I think it may be good to include the return statement description.